pkg/ebpf/tracee.go

```diff
@@ -1246,6 +1249,12 @@ func (t *Tracee) GetTailCalls() ([]*events.TailCall, error) {
 func (t *Tracee) attachProbes() error {
 	var err error
 
+	// attach control plane probes first
```

Comment:
Cool. This will, eventually, after attaching the control plane probes, attach all probe groups from all extensions!